packages/integration-tests/test/rpc.spec.ts

```diff
@@ -89,7 +88,7 @@ describe('Transactions', () => {
     const result = await signer.sendTransaction(tx)
     await result.wait()
 
-    // "from" is calculated client side here, so
+    // from is calculated client side here, so
```

Comment:
Any reason to remove the quotes around `from` here? I think it makes it easier to read so that the reader knows that `from` is a noun instead of a preposition which is how its more commonly used

Reply:
No real reason. I think I did this while making updating some strings to use single quotes consistently and didn't really know why this was quoted. I'll switch it back